userdocs/src/usage/eks-managed-nodes.md

```diff
@@ -7,19 +7,20 @@ An EKS managed node group is an autoscaling group and associated EC2 instances t
 **NEW** [Launch Template support for managed nodegroups](launch-template-support.md)
 
 !!!info
-    The term "unmanaged nodegroups" has been used to refer to nodegroups that eksctl has supported since the beginning and uses by default. The `ClusterConfig` file continues to use the `nodeGroups` field for defining unmanaged nodegroups, and a new field `managedNodeGroups` has been added for defining managed nodegroups.
+    The term "unmanaged nodegroups" has been used to refer to nodegroups that eksctl has supported since the beginning (represented via the `nodeGroups` field). The `ClusterConfig` file continues to use the `nodeGroups` field for defining unmanaged nodegroups, and managed nodegroups are defined with the ``managedNodeGroups` field.
```

Comment:
```suggestion
    The term "unmanaged nodegroups" has been used to refer to nodegroups that eksctl has supported since the beginning (represented via the `nodeGroups` field). The `ClusterConfig` file continues to use the `nodeGroups` field for defining unmanaged nodegroups, and managed nodegroups are defined with the `managedNodeGroups` field.
```

Reply:
Good catch.